utils/data_reader.py

```diff
@@ -58,15 +91,28 @@ def add_parser_db_args(parser):
 
 
 def case_insensitive_lookup(row, key, version):
-    desired_key = DATA_DICTIONARY[key][version]
+    defaults = {}
+    if type(version) == str:
+        desired_key = DATA_DICTIONARY[version][key]
+    else:
+        desired_key = version.get(key, key)
+        defaults = version.get("default_values", {})
```

Comment:
I'd rather see this logic added to it's own method instead of baking it into `case_insentitive_lookup`.  It would eliminate much of the complexity of this method and this specialized logic is only needed in one place (`translate_row` in `extract.py`).